reference/architecture.dita

```diff
@@ -10,4 +10,16 @@
   <shortdesc>DITA Open Toolkit is an open-source implementation of the OASIS specification for the Darwin Information
     Typing Architecture. The toolkit uses Ant, XSLT, and Java to transform DITA content (maps and topics) into different
     deliverable formats.</shortdesc>
+  <prolog>
+    <metadata>
+      <keywords>
+        <indexterm>authoring formats<indexterm>DITA</indexterm></indexterm>
+        <indexterm>OASIS</indexterm>
+        <indexterm>architecture<indexterm>components</indexterm></indexterm>
```

Comment:
With only 2 subentries that both point to the same page, a single “architecture” entry might suffice.